Dockerfile

```diff
@@ -1,23 +1,25 @@
-FROM node:carbon
+FROM node:8-alpine
 
 # Change ownership of the global node modules directory
 # to allow installs without requiring sudo privileges.
 #
 # See: https://docs.npmjs.com/getting-started/fixing-npm-permissions
-RUN ["bash", "-c", "chown -R node $(npm config get prefix)/{lib/node_modules,bin,share}"]
+RUN chown -R node $(npm config get prefix)
```

Comment:
bash is not included by default in alpine, therefore we cannot use the {} notation for specifying specific directories. I don't believe excluding the other directories made too much of a different anyhow, but I haven't actually benchmarked it. If you think it will make a significant difference, we can do a `chown` on each directory separately.